### ISSUE TITLE
Terraform validate -var and -var-file flags will be removed in a future version

### ISSUE THREAD

**Issue by brettcurtis** [newcontext-oss/kitchen-terraform#417]
Just starting to learn kitchen-terraform and inspec and loving the project so far - thanks! This is just a heads up in case you're not aware yet. This may be a new warning with terraform .13.

```none
$$$$$$ Validating the Terraform configuration files...       
Warning: The -var and -var-file flags are not used in validate. Setting them has no effect.      
These flags will be removed in a future version of Terraform.
Success! The configuration is valid, but there were some validation warnings as shown above.
```
I see this since I'm using a variable in my driver:

```yaml
driver:
  name: terraform
  variables:
    billing_id: <%= ENV['BILLING_ID'] %>
```

This is how I'm passing secrets from my github actions, not sure if there is a better way or not.

**Comment by aaron-lane** [comment]
Thanks for bringing this up, @brettcurtis! Looks like we'll have to consider some more backward compatibility details :D

**Comment by aaron-lane** [comment]
This was fixed by #452 🗝️
